m3_json_schema.json

```diff
@@ -5,11 +5,21 @@
   "title": "The M3 JSON Schema",
   "comment": "profile, classes and properties are required; mappings is not",
   "required": [
+    "schema_version",
     "profile",
     "classes",
     "properties"
   ],
   "properties": {
+
+    "schema_version": {
+      "$id": "#/properties/schema_version",
+      "type": "string",
+      "title": "Schema Version",
+      "description": "Version of the JSON Schema to validate against. Change this to make a new release.",
+      "pattern": "1.0.beta"
```

Comment:
@geekscruff just to check that I'm understanding, when we make a change to the schema, we would update the version number here (line 20)?  like: 
      "pattern": "1.1.beta"

Reply:
Yep.